patches/core/ungoogled-chromium/fix-building-without-safebrowsing.patch

```diff
@@ -184,198 +184,241 @@
    // Testing functions.
    ConnectorsManager* ConnectorsManagerForTesting();
  
---- a/chrome/browser/extensions/api/enterprise_reporting_private/context_info_fetcher.cc
-+++ b/chrome/browser/extensions/api/enterprise_reporting_private/context_info_fetcher.cc
-@@ -49,7 +49,6 @@ void ContextInfoFetcher::Fetch(ContextIn
+--- a/chrome/browser/enterprise/signals/context_info_fetcher.cc
++++ b/chrome/browser/enterprise/signals/context_info_fetcher.cc
+@@ -53,7 +53,6 @@ void ContextInfoFetcher::Fetch(ContextIn
        GetAnalysisConnectorProviders(enterprise_connectors::FILE_DOWNLOADED);
    info.on_bulk_data_entry_providers =
        GetAnalysisConnectorProviders(enterprise_connectors::BULK_DATA_ENTRY);
 -  info.realtime_url_check_mode = GetRealtimeUrlCheckMode();
    info.on_security_event_providers = GetOnSecurityEventProviders();
    info.browser_version = version_info::GetVersionNumber();
  
-@@ -91,18 +90,6 @@ std::vector<std::string> ContextInfoFetc
+@@ -79,11 +78,6 @@ std::vector<std::string> ContextInfoFetc
    return connectors_service_->GetAnalysisServiceProviderNames(connector);
  }
  
--api::enterprise_reporting_private::RealtimeUrlCheckMode
+-safe_browsing::EnterpriseRealTimeUrlCheckMode
 -ContextInfoFetcher::GetRealtimeUrlCheckMode() {
--  switch (connectors_service_->GetAppliedRealTimeUrlCheck()) {
--    case safe_browsing::REAL_TIME_CHECK_DISABLED:
--      return api::enterprise_reporting_private::
--          REALTIME_URL_CHECK_MODE_DISABLED;
--    case safe_browsing::REAL_TIME_CHECK_FOR_MAINFRAME_ENABLED:
--      return api::enterprise_reporting_private::
--          REALTIME_URL_CHECK_MODE_ENABLED_MAIN_FRAME;
--  }
+-  return connectors_service_->GetAppliedRealTimeUrlCheck();
 -}
 -
  std::vector<std::string> ContextInfoFetcher::GetOnSecurityEventProviders() {
    return connectors_service_->GetReportingServiceProviderNames(
        enterprise_connectors::ReportingConnector::SECURITY_EVENT);
+--- a/chrome/browser/enterprise/signals/context_info_fetcher.h
++++ b/chrome/browser/enterprise/signals/context_info_fetcher.h
+@@ -6,10 +6,10 @@
+ #define CHROME_BROWSER_ENTERPRISE_SIGNALS_CONTEXT_INFO_FETCHER_H_
+ 
+ #include <string>
++#include <memory>
```

Comment:
Why is this include needed?

Reply:
`std::unique_ptr` is used around line 57 which was previously getting access to it from one of `safe_browsing_prefs.h`'s includes.